test/common/upstream/load_balancer_fuzz_base.h

```diff
@@ -0,0 +1,74 @@
+#include "envoy/config/cluster/v3/cluster.pb.h"
+
+#include "common/upstream/load_balancer_impl.h"
+
+#include "test/common/upstream/load_balancer_fuzz.pb.validate.h"
+#include "test/mocks/common.h"
+#include "test/mocks/runtime/mocks.h"
+#include "test/mocks/upstream/cluster_info.h"
+#include "test/mocks/upstream/host_set.h"
+#include "test/mocks/upstream/load_balancer_context.h"
+#include "test/mocks/upstream/priority_set.h"
+
+namespace Envoy {
+
+namespace Random {
+class FakeRandomGenerator : public RandomGenerator {
+public:
+  FakeRandomGenerator() = default;
+  ~FakeRandomGenerator() override = default;
+
+  // RandomGenerator
+  uint64_t random() override; // returns a part of the generated bytestring
+  std::string uuid() override { return ""; }
+  uint32_t counter = 0; // Will get modded against byte string length for byte string index
+  std::vector<uint64_t> bytestring_; // String of bytes that returns on random() calls
+};
+} // namespace Random
+
+namespace Upstream {
+
+// This class implements replay logic, and also handles the initial setup of static host sets and
+// the subsequent updates to those sets.
+class LoadBalancerFuzzBase {
+public:
+  LoadBalancerFuzzBase()
+      : stats_(ClusterInfoImpl::generateStats(stats_store_)){
+
+        };
```

Comment:
I'm surprised the formatting tool didn't catch/fix this...?

Reply:
Yeah, I run format, weird it didn't catch, Asra mentioned this as well in #13400.

Reply:
Changed to {} in that PR.